pandas/tests/io/test_parquet.py

```diff
@@ -1131,6 +1131,33 @@ def test_infer_string_large_string_type(self, tmp_path, pa):
     #     assert result["strings"].dtype == "string"
     # FIXME: don't leave commented-out
 
+    def test_non_nanosecond_timestamps(self, tmp_path, pa):
+        # GH#49236
+        #
+        # pandas 1.x didn't support non-nanosecond datetimes.
```

Comment:
While the comments here are well intentioned, I would just keep the GH# comment and remove all of the rest. 

Generally if a developer wanted to see more discussion they could check back on github from there. We have a rather large code base as is so try to keep comments at a minimum - otherwise there could be quite a lot to read :-)

Reply:
Removed